tests/validators/test_function.py

```diff
@@ -52,11 +52,11 @@ def f(input_value, **kwargs):
         v.validate_python('12345')
     assert exc_info.value.errors() == [
         {
-            'kind': 'str_too_long',
+            'kind': 'too_long',
             'loc': [],
             'message': 'String must have at most 5 characters',
             'input_value': '12345x',
-            'context': {'max_length': 5},
+            'context': {'type': 'String', 'max_length': 5, 'element_name': 'characters'},
```

Comment:
@samuelcolvin I am not sure if this 'element_name' (I fully expect you to not like that context variable and ask me to rename it :D) should be printed out. It's kinda redundant? Same goes for the `type` in this context, I guess. It's all explicit in the error's message anyway.

Reply:
Part of the point of context is to allow translation of errors in downstream packages.

So type should be included.